src/useradd.c

```diff
@@ -2257,6 +2258,27 @@ static void create_mail (void)
 	}
 }
 
+static void check_uid_range(int rflg, uid_t user_id)
+{
+	uid_t uid_min ;
+	uid_t uid_max ;
+	if(rflg){
+		uid_min = (uid_t)getdef_ulong("SYS_UID_MIN",101UL);
+		uid_max = (uid_t)getdef_ulong("SYS_UID_MAX",getdef_ulong("UID_MIN",1000UL)-1);
+		if(uid_min <= uid_max){
+			if(user_id < uid_min || user_id >uid_max)
+				printf(_("warning: %s's uid %d outside of the SYS_UID_MIN %d and SYS_UID_MAX %d range.\n"),user_name, user_id, uid_min, uid_max);
```

Comment:
I've already merged this, but now it occurred to me - should these go to stderr instead?

Reply:
> 
> 
> I've already merged this, but now it occurred to me - should these go to stderr instead?

@hallyn 
I don't think so.  This is a warning other than a error.

Reply:
No, warnings should go to stderr.

Reply:
> 
> 
> No, warnings should go to stderr.

ok, I will prepare a patch for this.